### PR TITLE
Adding a simple data structure 

### PR DESCRIPTION
**Pull request by panzors**
My first pull request

### CODE REVIEW

**Comment by peabnuts123** [comment]
This is where we code review

File: BountyBoard.Core/AchievementManagement.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+{
+    public class AchievementManagement: CrudLink<Achievement>
+    {
+        public AchievementManagement(IDatabaseContext context) : base(context)
+        {
+
+        }
+
+        
+
```
Comment (peabnuts123) [inline BountyBoard.Core/AchievementManagement.cs:18]: sweet file bro

File: BountyBoard.Core/Data/Achievement.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BountyBoard.Core.Data
+{
+    public class Achievement : DatabaseObject
+    {
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public byte[] Image { get; set; }
```
Comment (peabnuts123) [inline BountyBoard.Core/Data/Achievement.cs:13]: Can image be represented in a better way than `byte[]`?

File: BountyBoard.Core/IDatabaseContext.cs [hunk 3]
```diff
@@ -0,0 +1,6 @@
+﻿namespace BountyBoard.Core
+{
+    public interface IDatabaseContext
+    {
+    }
+}
```
Comment (peabnuts123) [inline BountyBoard.Core/IDatabaseContext.cs:6]: Sick file

**Comment by peabnuts123** [comment]
Lots of empty files in these commits. Can you justify?

File: BountyBoard.Core/Data/Achievement.cs (hunk 2 above)
Comment (panzors) [inline BountyBoard.Core/Data/Achievement.cs:13]: in the database, i've found that it's better leaving it as a byte[]. i'm open to ideas

**Comment by panzors** [comment]
Most of these are scaffolds for a preview of how the system's going to look like. I'm still working on filling a lot of these files in.

**Comment by peabnuts123** [comment]
Alright, looks good.

**Pull request merged**